src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/action/TransitionsActionConfig.kt

```diff
@@ -22,7 +22,10 @@ import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.Transiti
 import org.elasticsearch.client.Client
 import org.elasticsearch.cluster.service.ClusterService
 
-data class TransitionsActionConfig(val transitions: List<Transition>) : ActionConfig(ActionType.TRANSITION, null, null) {
+const val TRANSITION_ACTION_INDEX = -1
+data class TransitionsActionConfig(
+    val transitions: List<Transition>
+) : ActionConfig(ActionType.TRANSITION, null, null, TRANSITION_ACTION_INDEX) {
```

Comment:
np:  I think it is ok to just pass in the `index` as `-1` instead of creating this `const val` outside of the class.